### DISCUSSION TITLE
How to swap on Base Chain?

### DISCUSSION THREAD

**Discussion by jhcdx9999** [uniswap-python/uniswap-python#386] [Q&A]
I would like to know how to swap on Base Chain, for example swapping ETH to 0x72499bddb67f4ca150e1f522ca82c87bc9fb18c8. If you can solve this problem by showing your codes, leave your Base wallet address below your answer. I would be very glad to buy you some cups of coffee :D

**Comment by fsaint** [comment]
I added base into [this fork/branch ](https://github.com/fsaint/uniswap-python/tree/fsaint/base) and was able to use get_token_balance and make_trade on the base network. Haven't tested other functionality so be careful.
